src/Driver/OCI8/Statement.php

```diff
@@ -101,7 +101,7 @@ public function bindParam($param, &$variable, $type = ParameterType::STRING, $le
 
             assert($lob !== false);
 
-            $lob->writetemporary($variable, OCI_TEMP_BLOB);
+            $lob->writeTemporary($variable, OCI_TEMP_BLOB);
```

Comment:
After the bump, PHPStan started to complain about this line. I've pushed a commit to make PHPStan happy again.

Reply:
Yes, probably. Thanks for the hint.